code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -289,20 +289,28 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 /mob/living/carbon/human/proc/update_markings()
 	//Reset our markings.
 	remove_overlay(MARKINGS_LAYER)
+	remove_overlay(HAND_MARKINGS_LAYER)
 
 	//Base icon.
 	var/icon/markings_standing = icon("icon" = 'icons/mob/body_accessory.dmi', "icon_state" = "accessory_none_s")
+	var/icon/hand_markings = icon("icon" = 'icons/mob/body_accessory.dmi', "icon_state" = "accessory_none_s")
 
 	//Body markings.
 	var/obj/item/organ/external/chest/chest_organ = get_organ("chest")
-	if(chest_organ && m_styles["body"])
+	if(chest_organ && m_styles["body"] != "None")
 		var/body_marking = m_styles["body"]
 		var/datum/sprite_accessory/body_marking_style = GLOB.marking_styles_list[body_marking]
 		if(body_marking_style && body_marking_style.species_allowed && (dna.species.name in body_marking_style.species_allowed))
 			var/icon/b_marking_s = icon("icon" = body_marking_style.icon, "icon_state" = "[body_marking_style.icon_state]_s")
 			if(body_marking_style.do_colouration)
 				b_marking_s.Blend(m_colours["body"], ICON_ADD)
 			markings_standing.Blend(b_marking_s, ICON_OVERLAY)
+
+			if(get_organ("l_hand") || get_organ("r_hand"))
```

Comment:
Have you tested this with characters missing a hand?

Reply:
This will sound bad, but there's no point in checking for that, markings don't care if the limb they're on is missing or not, in this PR, and before it.
![image](https://user-images.githubusercontent.com/71735193/202792913-1b8b95f5-4ee8-448d-86dd-bc954ba0a01e.png)

![image](https://user-images.githubusercontent.com/71735193/202792988-f2f9f607-92f0-46cd-b6b9-395cd994d770.png)

Reply:
That is /so/ scuffed, but definitely out of scope